suite-native/module-device-onboarding/src/hooks/useExitAlert.ts

```diff
@@ -24,7 +24,7 @@ type NavigationProps = StackToStackCompositeNavigationProps<
     DeviceOnboardingStackRoutes,
     RootStackParamList
 >;
-export const useExitAlert = () => {
+export const useExitAlert = (handleContinueButtonPress?: () => void) => {
```

Comment:
nit:  I would prefer `on` prefix since the whole repo follows this convention. (https://medium.com/@master_43681/understanding-and-proper-usage-of-on-and-handle-prefixes-in-react-function-naming-119f0beea400)
```suggestion
export const useExitAlert = (onContinueButtonPress?: () => void) => {
```

Reply:
But inside useExitAlert, it is a handler, it's not a named property of the hook, so I believe it follows the convention you've linked.